core/types/state_account_marshalling.go

```diff
@@ -73,24 +73,31 @@ func (s *StateAccount) Hash() (*big.Int, error) {
 // MarshalFields, the bytes scheme is:
 // [0:32] Nonce uint64 big-endian in 32 byte
 // [32:64] Balance
-// [64:96] Root
-// [96:128] CodeHash
+// [64:96] CodeHash
+// [96:128] Root
+// [128:160] KeccakCodeHash
+// [160:192] CodeSize
```

Comment:
TBD: Consider optimizing this. Do we need to store an uint64 in 32 bytes or can we pack these more tightly? What do you think? @noel2004

Reply:
I think both way should be ok. A more tightly scheme save ~10% space for account data so it may be resonable.

Reply:
However, currently we just marshal the data as array of fields and we need some additional entries or notes to do more compression.

Reply:
Is there a reason why this structure specifically?

Reply:
No special reason, do you want to suggest a different marshalling scheme?

Reply:
If `Nonce` and `CodeSize` both are 8 bytes each, I was wondering why not pack them together in the first (or last) 32 bytes, instead of using 32 bytes for each of them individually?

Reply:
I agree with @roynalnaruto on the packing idea. It can reduce the storage size by 17%

Reply:
I also agree. All we need is just an additional parsing when reading the data into an `AccountData` strcture

Reply:
Made the changes, pending on https://github.com/scroll-tech/zktrie/pull/5.